bin/import_swc.py

```diff
@@ -0,0 +1,447 @@
+#!/usr/bin/env python
+'''
+Downloads and prepares (parts of) the "Spoken Wikipedia Corpora" for DeepSpeech.py
+Use "python3 import_swc.py -h" for help
+'''
+from __future__ import absolute_import, division, print_function
+
+# Make sure we can import stuff from util/
+# This script needs to be run from the root of the DeepSpeech repository
+import os
+import sys
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+
+import re
+import csv
+import sox
+import wave
+import shutil
+import random
+import tarfile
+import argparse
+import progressbar
+import unicodedata
+import xml.etree.cElementTree as ET
+
+from os import path
+from glob import glob
+from collections import Counter
+from multiprocessing.pool import ThreadPool
+from util.text import Alphabet, validate_label
+from util.downloader import maybe_download, SIMPLE_BAR
+
+SWC_URL = "https://www2.informatik.uni-hamburg.de/nats/pub/SWC/SWC_{language}.tar"
+SWC_ARCHIVE = "SWC_{language}.tar"
+LANGUAGES = ['dutch', 'english', 'german']
+FIELDNAMES = ['wav_filename', 'wav_filesize', 'transcript']
+CHANNELS = 1
+SAMPLE_RATE = 16000
+AUDIO_PATTERN = 'audio*.ogg'
+WAV_NAME = 'audio.wav'
+ALIGNED_NAME = 'aligned.swc'
+
+SUBSTITUTIONS = {
+    'german': [
+        (re.compile(r'\$'), 'dollar'),
+        (re.compile(r'€'), 'euro'),
+        (re.compile(r'£'), 'pfund'),
+        (re.compile(r'ein tausend ([^\s]+) hundert ([^\s]+) er( |$)'), r'\1zehnhundert \2er '),
+        (re.compile(r'ein tausend (acht|neun) hundert'), r'\1zehnhundert'),
+        (re.compile(r'eins punkt null null null punkt null null null punkt null null null'), 'eine milliarde'),
+        (re.compile(r'punkt null null null punkt null null null punkt null null null'), 'milliarden'),
+        (re.compile(r'eins punkt null null null punkt null null null'), 'eine million'),
+        (re.compile(r'punkt null null null punkt null null null'), 'millionen'),
+        (re.compile(r'eins punkt null null null'), 'ein tausend'),
+        (re.compile(r'punkt null null null'), 'tausend'),
+        (re.compile(r'punkt null'), None)
+    ]
+}
+
+DONT_NORMALIZE = {
+    'german': 'ÄÖÜäöüß',
+    'dutch': 'Ĳĳ'
```

Comment:
At least to https://en.wikipedia.org/wiki/IJ_(digraph) it seemed to be complicated. So I kept it...

Reply:
If the digraph needs to be its own character, should we then normalize IJ (separate letters) to the single character version? Or does that never happen in the corpus? Otherwise you can have Ĳfoobar and IJfoobar correspond to different transcripts (the former uses the ligature the latter uses I and J).